src/test/java/com/cloudant/tests/DatabaseTest.java

```diff
@@ -71,6 +72,22 @@ public void permissions() {
 
     }
 
+    /**
+     * Test that when called against a DB that is not a Cloudant service
+     * an UnsupportedOperationException is thrown
+     */
+    @Test
+    public void testPermissionsException() {
+        try {
+            Map<String, EnumSet<Permissions>> userPerms = db.getPermissions();
+            //ignore the test in cases where we don't get an exception
+            Assume.assumeTrue(false);
+        }catch(UnsupportedOperationException e){

```

Comment:
You wouldn't need to Assume using `expected` because you'd remove that code and if the exception wasn't thrown it would fail. which is what the Assume.assumeTrue is effectively doing